### ISSUE TITLE
Call to undefined method Image::applyFilter()

### ISSUE THREAD

**Issue by vdcrea** [DeuxHuitHuit/image_upload#35]
It seems there is an issue, using the latest integration branch of JIT, I guess it is due to the latests updates of JIT.

Call to undefined method Image::applyFilter()  
/extensions/image_upload/fields/field.image_upload.php around line 60

**Comment by nitriques** [comment]
> the latest integration branch of JIT,

You mean version 2.0.0 ?

**Comment by vdcrea** [comment]
Yes, using JIT 2.0.2. I tryied to change applyFilter() to applyFilterToImage() (if i remember well), also tryied to require_once other classes than Image but I didn't find the solution.

**Comment by nitriques** [comment]
EDIT: sorry wrong project.

Yeah the resize option is not fully compatible with JIT 2.0.0
